WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderFragment.kt

```diff
@@ -107,66 +112,66 @@ class ReaderFragment : Fragment(R.layout.reader_fragment_layout), ScrollableView
         }
     }
 
-    private fun initToolbar() {
+    private fun initToolbar(binding: ReaderFragmentLayoutBinding) = with(binding) {
```

Comment:
I think I like a bit more the approach with `ReaderFragmentLayoutBinding.initToolbar()`. It allows you to call the `with` just once in the `onViewCreated`. Let me know what you think 👍

Reply:
I totally agree. This has been addressed in 822c94a.